roofit/roofit/perf/RooFitBinnedBenchmarks.cxx

```diff
@@ -0,0 +1,147 @@
+#include "RooWorkspace.h"
+#include "RooAddPdf.h"
+#include "RooRealVar.h"
+#include "RooMinimizer.h"
+#include "TFile.h"
+#include "TH1.h"
+#include "TError.h"
+#include "RooStats/HistFactory/Measurement.h"
+#include "RooStats/HistFactory/HistoToWorkspaceFactoryFast.h"
+#include "RooStats/ModelConfig.h"
+
+#include "benchmark/benchmark.h"
+
+using namespace RooFit;
+using namespace RooStats;
+using namespace HistFactory;
+
+void buildBinnedTest()
```

Comment:
We probably should implement a benchmark fixture and move the initialization (creating the workspaces) and teardown there.